app/models/record.rb

```diff
@@ -3,8 +3,15 @@ class Record < ApplicationRecord
 
   include SearchScope
   belongs_to :register
-  scope :current, -> { where(Arel.sql("data->> 'end-date' is null")) }
-  scope :archived, -> { where(Arel.sql("data->> 'end-date' is not null")) }
+  scope :current, -> {
+    where(Arel.sql("data->> 'end-date' is null or data->> 'end-date' >= '#{Date.today}'"))
```

Comment:
I think this should be `>` rather than `>=` as would would consider one with an end date of today to be archived.

Reply:
Okay - I've put archived to be `<=` instead of `<` so that a day won't be missed two queries.